test/controllers/register/establishers/individual/PostCodeLookupControllerSpec.scala

```diff
@@ -87,7 +110,7 @@ class PostCodeLookupControllerSpec extends ControllerSpecBase with MockitoSugar
 
       val result = controller().onSubmit(NormalMode, firstIndex)(postRequest)
 
-      status(result) mustBe BAD_REQUEST
+      status(result) mustBe OK
```

Comment:
Amend test description to reflect now an ok not a bad request

Reply:
@AndrewArkless Changed that for you